tests/anthropic/test_anthropic_autolog.py

```diff
@@ -0,0 +1,55 @@
+from unittest.mock import patch
+
+import anthropic
+from anthropic.types import Message, TextBlock, Usage
+
+import mlflow.anthropic
+from mlflow.entities.span import SpanType
+
+from tests.tracing.helper import get_traces
+
+DUMMY_CREATE_MESSAGE_REQUEST = {
+    "model": "test_model",
+    "max_tokens": 1024,
+    "messages": [{"role": "user", "content": "test message"}],
+}
+
+DUMMY_CREATE_MESSAGE_RESPONSE = Message(
+    id="test_id",
+    content=[TextBlock(text="test answer", type="text")],
+    model="test_model",
+    role="assistant",
+    stop_reason="end_turn",
+    stop_sequence=None,
+    type="message",
+    usage=Usage(input_tokens=10, output_tokens=18),
+)
+
+
+def create(self, max_tokens, model, messages):
+    return DUMMY_CREATE_MESSAGE_RESPONSE
+
+
+def test_messages_autolog():
+    with patch("anthropic.resources.Messages.create", new=create):
+        mlflow.anthropic.autolog()
```

Comment:
```suggestion
    mlflow.anthropic.autolog()
    with patch("anthropic.resources.Messages.create", new=create):
```

because `mlflow.anthropic.autolog()` doesn't need `anthropic.resources.Messages.create` to be mocked.

Reply:
as `mlflow.anthropic.autolog()` also patches `Messages.create` method, mocking the method after calling autolog() will disable tracing